lang/ru/lang.php

```diff
@@ -16,17 +16,27 @@
         'access_posts' => 'Управление записями блога',
         'access_categories' => 'Управление категориями блога',
         'access_other_posts' => 'Управление записями других пользователей',
+        'access_import_export' => 'Разрешено импортировать и экспортировать записи',
+        'access_publish' => 'Разрешено публиковать записи',
+        'manage_settings' => 'Управление настройками блога',
         'delete_confirm' => 'Вы уверены, что хотите сделать это?',
         'chart_published' => 'Опубликовано',
         'chart_drafts' => 'Черновики',
         'chart_total' => 'Всего',
+        'settings_description' => 'Управление настройками блога',
+        'show_all_posts_label' => 'Показывать все записи для внутренних (бэкенд) пользователей',
+        'show_all_posts_comment' => 'Показывать опубликованные и неопубликованные записи на фронтенде для внутренних (бэкенд) пользователей',
+        'tab_general' => 'Основные'
         'new_category' => 'Новая категория'
```

Comment:
This breaks the language file, please make sure ALL array elements have a trailing comma. Also, please test in the plugin in a running OctoberCMS instance too.

Reply:
Sorry for the inattention.
Corrected.